components/CustomButton.js

```diff
@@ -94,4 +94,54 @@ const CustomButton = ({
     );
 };
 
+const CustomButton2 = ({
+    text,
+    color,
+    warningText,
+    minHeight,
+    maxHeight,
+    onPress,
+    displayIcon,
+    horizontalMargin,
+    isAllCap,
+    fixedTextWidth,
+    fontWeight,
+    fontSize,
+    lineHeight,
+    height,
+    marginVertical,
+    ...others
+}) => {
+    return (
+        <ButtonContainer
+            color={others.disabled ? 'grey' : color}
+            onPress={onPress}
+            disabled={others.disabled}
+            maxHeight={maxHeight}
+            height={height}
+            minHeight={minHeight}
+            horizontalMargin={horizontalMargin}
+            marginVertical={marginVertical}
+        >
+            <StandardTextContainer>
+                <SmallerStandardText
+                    color={color}
+                    isDisable={others.disabled}
+                    fixedTextWidth={fixedTextWidth}
+                >
+                    {isAllCap ? text.toUpperCase() : text}
+                </SmallerStandardText>
+            </StandardTextContainer>
```

Comment:
The only diff I see between `CustomButton2` and `CustomButton` is that they use different `Text`. This is probably not a good reason to split it into a new component, or we will end up with a bunch of buttons to maintain. Try adding a parameter to the Custombutton to make it either use `SmallerStandardText` instead of `StandardText` rather than making a new button component.

Reply:
Something along this line where `isSmallText` is a new variable. With this, anywhere you wanna call it with small text, do `isSmallText=True`. I am sure there are examples like this everywhere. (I haven't tried the code below yet, there might be a shorter version of if)
```
{isSmallText ? 
    <SmallerStandardText
        color={color}
        isDisable={others.disabled}
        fixedTextWidth={fixedTextWidth}
    >
        {isAllCap ? text.toUpperCase() : text}
    </SmallerStandardText> : 
    <StandardText
          color={color}
          isDisable={others.disabled}
          fixedTextWidth={fixedTextWidth}
      >
          {isAllCap ? text.toUpperCase() : text}
      </StandardText>
```